src/core/pipelines/manifest/index.ts

```diff
@@ -91,24 +90,37 @@ export default function createManifestPipeline(
    * @returns {Observable}
    */
   return function fetchManifest(url : string) : Observable<Manifest> {
-    return loader({ url }).pipe(
+    /**
+     * Request content for URL, and a given content type.
+     * Return the response.
+     * @param {string} _url
+     * @param {string} contentType
+     * @returns {Observable}
+     */
+    function loadContent<T extends string|Document>(
+      _url: string,
+      contentType?: "text"|"document"
```

Comment:
A little problematic to rely on `contentType` here, as coming from the pipelines it seems kind of a random property to give to a loader (which should know what it's talking about).

IMO, the pipelines does not know what those are, so it should not infer it.
But the parser does.

We should think a little more about it. Like: Does it makes sense to go through the manifest.loader again when loading sub-parts? It's always a different logic (we ignore a possible customLoader in the latter case)